Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rust:1.59-slim AS builder
+
+WORKDIR /build
+RUN apt-get update
+RUN apt-get install -y git clang cmake libsnappy-dev
+COPY . .
+RUN cargo build --release --package rendezvous-server --bin rendezvous-server
+
+
+FROM debian:bullseye-slim
```

Comment:
Using `distroless` will likely make for a smaller image: https://github.com/GoogleContainerTools/distroless/blob/main/examples/rust/Dockerfile

Reply:
Thanks! Will try that